config.yml

```diff
@@ -0,0 +1,56 @@
+# GitLab user. git by default
+user: git
+
+# Url to gitlab instance. Used for api calls. Should end with a slash.
+# Default: http://localhost:8080/
+# You only have to change the default if you have configured Unicorn
+# to listen on a custom port, or if you have configured Unicorn to
+# only listen on a Unix domain socket.
+gitlab_url: "http://192.168.59.103:8080/box/api/gls"
```

Comment:
We'll likely need to change this to point to some service rather than a hardcoded IP. @david-martin would this be the gitlab-shell service or a different one?